markdownlint/TOP010_useLazyNumbering/TOP010_useLazyNumbering.js

```diff
@@ -1,36 +1,29 @@
 module.exports = {
   names: ["TOP010", "ordered-list-for-lazy-numbering"],
-  description: "Ordered lists should use lazy numbering",
+  description: "Ordered lists must always use 1. as a prefix (lazy numbering)",
   information: new URL(
     "https://github.com/TheOdinProject/curriculum/blob/main/markdownlint/docs/TOP010.md"
   ),
-  tags: ["lazy-numbering"],
+  tags: ["ol"],
   parser: "markdownit",
   function: function TOP010(params, onError) {
     params.parsers.markdownit.tokens.forEach((token) => {
-      if (
-        token.tag === "li" &&
-        token.line !== undefined &&
-        /\d/.test(token.line) &&
-        !/^\s*1\./.test(token.line)
-      ) {
-        const originalText = token.line;
-        // https://regexr.com/80oan to test this regex
-        const updatedText = originalText.replace(/^\s*\d+\./, (match) =>
-          match.replace(/\d+/, "1")
-        );
-        const textLength = token.line.length;
-        const lineNumber = token.lineNumber;
+      // https://regexr.com/80oan to test this regex
+      const digit = /\d/;
```

Comment:
I was concerned about this as well, but never properly checked it. My concern was not only double digits but also in the event we receive double digits with a leading 1. I'll double check the regex and make sure it accounts for that stuff as well!

Reply:
`10.` etc should still trigger since `token.info` would be `'10'`

We're very unlikely to encounter this anyway. I've an open PR fixing the 4-5 only lessons that have double digit prefixes still, so we'd only encounter double digits in new lessons/changes, which we can fix in review. It's just a funky part of the CommonMark spec that requires more indentation for nested lists if an OL item prefix has more than one digit